src/Symfony/Component/Notifier/Recipient/Recipient.php

```diff
@@ -35,8 +58,25 @@ public function email(string $email): self
         return $this;
     }
 
-    public function getEmail(): string
+    public function hasPhone(): bool
     {
-        return $this->email;
+        return '' !== $this->phone;
+    }
+
+    public function getPhone(): string
+    {
+        return $this->phone;
+    }
+
+    /**
+     * Sets the phone number (no spaces, international code like in +3312345678).
+     *
+     * @return $this
+     */
+    public function phone(string $phone): self
```

Comment:
Does it makes sense adding this method to the interface and the email() method too?

Just asking

Reply:
I also thought about this. But these methods are actually not used by the current internal implementations (e.g. Channel classes). They are just convenience methods for the default recipient implementation and shouldn't be in the interface for that reason imho.